SPDX-License-Identifier: BSD-3-Clause

Change build-type from Custom to Simple so that entropy's custom setup
script does not run its platform-specific C probes. On OpenBSD, those
probes include <sys/random.h> and <linux/random.h>, neither of which
exist.

Index: entropy.cabal
--- entropy.cabal.orig
+++ entropy.cabal
@@ -17,7 +17,7 @@ homepage:       https://github.com/TomMD
 bug-reports:    https://github.com/TomMD/entropy/issues
 stability:      stable
 
-build-type:     Custom
+build-type:     Simple
 
 tested-with:
   GHC == 9.14.1
@@ -47,13 +47,6 @@ Flag DoNotGetEntropy
   Default: False
   Manual: True
 
-custom-setup
-  setup-depends: Cabal >= 1.10 && < 3.17
-               , base < 5
-               , filepath < 1.6
-               , directory < 1.4
-               , process < 1.7
-
 library
   ghc-options:  -O2
   exposed-modules: System.Entropy
